From: Chun-Min Chang <chun.m.chang@gmail.com>
Date: Tue, 5 Dec 2023 20:08:00 +0000
Subject: Bug 1864008 - Move libvpx to libgkcodecs, part 2
 r=webrtc-reviewers,padenot,mjf

This patch addes trampoline headers for libvpx.

To follow the libwebrtc merge procedure, the vpx headers are silently
replaced with "trampoline" headers, which do nothing but include real
VPX headers. This makes the libwebrtc-merge process easier without
worrying headers' paths.

On the other hand, the `rtc_build_libvpx` is set to `true` in
webrtc.gni, so moz.build file for third_party/libvpx can be generated by
the gn_processor in the next patch.

Differential Revision: https://phabricator.services.mozilla.com/D195495
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/d43978d3d8356e176fac2ad18f328871f36698ce
---
 webrtc.gni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/webrtc.gni b/webrtc.gni
index 931a0a24e5..d763691759 100644
--- a/webrtc.gni
+++ b/webrtc.gni
@@ -286,7 +286,7 @@ declare_args() {
   # Disable these to not build components which can be externally provided.
   rtc_build_json = !build_with_mozilla
   rtc_build_libsrtp = !build_with_mozilla
-  rtc_build_libvpx = !build_with_mozilla
+  rtc_build_libvpx = true
   rtc_libvpx_build_vp9 = true
   rtc_build_opus = !build_with_mozilla
   rtc_build_ssl = !build_with_mozilla
